Need to make copy of keys using during containerized installation (#184)
SUSE/doc-unversioned: trento/migration/container-install.md

1. Create the secret environment variables:

>    > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust upcoming commands for env file usage. Regardless, make sure you keep a copy of the generated keys in a safe location, in case you need to reuse them in the future. 

Reviewer: ```suggestion
   > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust upcoming commands for env file usage. In any case, make sure you keep a copy of the generated keys in a safe location, in case you need to reuse them in the future. 
```

>    > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust upcoming commands for env file usage. In any case, make sure you keep a copy of the generated keys in a safe location, in case you need to reuse them in the future. 
> 

Reviewer: ```suggestion
   > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust the docker command below for use with the env file. In any case, make sure you keep a copy of the generated keys in a safe location, in case you need to reuse them in the future. 

```

>    > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust upcoming commands for env file usage. Regardless, make sure you keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future. 

Reviewer 2: What do you think about shortening it a bit up ?
Examples:
- Keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future.
- Additionally, keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future.
- Regardless keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future.

Author: Any of your suggestions LGTM. Pick up the one you like the most :-)

>    > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust upcoming commands for env file usage. Regardless, make sure you keep a copy of the generated keys in a safe location, in case you need to reutilize them in the future. 

Reviewer: ```suggestion
   > Note: Consider using an environment variable file, learn more about from [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env). Adjust upcoming commands for env file usage. Regardless, make sure you keep a copy of the generated keys in a safe location, in case you need to reuse them in the future. 
```

Reviewer: I'm not sure what "Regardless" refers to. Regardless what?
